tradingstrategy/client.py

```diff
@@ -7,17 +7,19 @@
 - :py:class:`Client` class
 
 """
-
+import datetime
 import os
 import tempfile
 import time
 from functools import wraps
 from json import JSONDecodeError
-from typing import Optional
+from typing import Optional, Set
 from pathlib import Path
 import logging
 
 # TODO: Must be here because  warnings are very inconveniently triggered import time
+import pandas as pd
+from backtrader import List
```

Comment:
I have a feeling this is the wrong (automatic) import, should it be from `typing`?